.github/actions/configure_dbt_environment/action.yaml

```diff
@@ -11,16 +11,12 @@ runs:
           echo "On pull request branch, setting dbt env to CI"
           {
             echo "TARGET=ci";
-            echo "CACHE_KEY=pr-caches/$GITHUB_HEAD_REF";
```

Comment:
`CACHE_KEY` is moving to the `dbt.env` file, since it's no longer different between master and PR environments.